Woof/Woof/View/OwnerFlow/OwnerWallkingsView/OwnerWalkingsView.swift

```diff
@@ -4,9 +4,11 @@ import SwiftUI
 struct OwnerWalkingsView: View {
     var body: some View {
         NavigationView {
-            Text(walkingsStubTitle)
-                .navigationTitle(navigationTitle)
-                .navigationBarTitleDisplayMode(.inline)
+//            Text(walkingsStubTitle)
+//                .navigationTitle(navigationTitle)
+//                .navigationBarTitleDisplayMode(.inline)
+
+            OwnerOnboardingView()
```

Comment:
Why do you put this `OwnerOnboardingView` here?
For me, those kinds of code shouldn't be pushed and merged in Main.
I can use a preview in canvas to see how it will look likes or you can add instructions on how to add this view to check it on a simulator.

Reply:
Sorry, the preview was buggy so I used this preview option
Deleted with 763777f